Bio/Phylo/Consensus.py

```diff
@@ -443,7 +443,7 @@ def _part(clades):
 
 
 def _sub_clade(clade, term_names):
-    """Extract a compatible subclade that only contains the given terminal names"""
+    """Extract a compatible subclade that only contains the given terminal names."""
```

Comment:
PRIVATE?

Reply:
Ideally, but it is already a very long line.